configure.ac

```diff
@@ -346,6 +346,45 @@ if test x$with_openssl != xno; then OSTREE_FEATURES="$OSTREE_FEATURES openssl";
 AM_CONDITIONAL(USE_OPENSSL, test $with_openssl != no)
 dnl end openssl
 
+dnl begin gnutls
```

Comment:
There's currently no reason to link to more than one of (openssl, gnutls, libgcrypt) right?  So maybe we should instead have `--with-crypto=` ?  And then the default value would be `glib`.

Reply:
I can try to look into this, but I'm not so great on autotools...

Reply:
No worries, I can do it.